src/code/getSpeciesSteadyState.R

```diff
@@ -0,0 +1,119 @@
+
+#' Get the steady-state values of species. The steady-state is considered to be the last value of the simulation with sufficiently
+#' long simulation time, i.e., where the rates of the processes do not change.
+#'
+#' @param speciesNames Vector of species names which steady-state values will be returned. By default, values of all species are returned. This also includes
+#' values of "state variable" parameters, as these are treated like species.
+#' @param steadyStateTime Simulation time (minites). Must be long enough for system to reach a steady-state. Default is 1000 minutes.
+#' @param ignoreIfFormula If "TRUE", species with initial values defined by formula are not included. Default is "TRUE".
+#' @param DCI_Info The DCI Info structure containing the DCI handle and all settings.
+#'
+#' @return A list containing two data frames - moleculesSteadyState and parametersSteadyState. The data frame moleculesSteadyState contains columns
+#' "Container Path", "Molecule Name", "Is Present", "Value", "Units", "Scale Divisor", "Neg. Values Allowed".
+#' The data frame parametersSteadyState. contains columns
+#' "Container Path", "Parameter Name", "Value", "Units".
+#' All species with the unit "�mol" that are located within the "Organism"-structure are treated as molecules and stored in the moleculesSteadyState-data frame.
+#' That means that local state variable parameters with the dimension "Amount" are treated as molecules.
+#' All species with units other then "�mol" OR the top container being not "Organism" are stored in the parametersSteadyState-data frame.
```

Comment:
>All species with the unit "µmol" that are located within the "Organism"-structure are treated as molecules 
>All species with units other then "µmol" OR the top container being not "Organism" are stored in the parametersSteadyState...

Are those reasonable assumptions? E.g. there are - even in "standard" models - molecule amounts which are not located under Organism (e.g. solid/insoluble drug located under Applications)

Reply:
It is indeed not a perfect solution, but I do not know of any other way to distinguish between species and state-variable parameters.

When working from the toolbox, it does not make any difference - all entries returned by "getSpeciesSteadyState()" will be species. The difference between "Molecule" and "Parameter" is only important when exporting the steady-state to xls and importing in MoBi.
Special cases, when molecules are located outside of "Organism", must be treated by the user manually - and the documentation describes it clearly.

As far as I know, amounts of molecules in Applications do not really matter, so it OK not to treat them properly.

But, yes, if there is another way to differentiate between "Molecule" and "Parameter", it should be used.

Reply:
OK, just checked it. You are right, currently there is no another possibility.
State variable parameters are defined as variables already in the simulation XML